manageiq-providers-oracle_cloud.gemspec

```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_dependency "oci", "~> 2.18"
+  spec.add_dependency "oci", "~> 2.19.0"
```

Comment:
Per https://github.com/oracle/oci-ruby-sdk/pull/72#discussion_r1353071148 oci-ruby-sdk doesn't follow semver and will introduce breaking changes in minor versions.

Reply:
Too bad about that response but it's better to know so we can be more careful.  Upgrades will be harder since we won't be updating and catching these things are quickly.  😢